homework/hw03-[1].py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+pip install mrjob
+
+
+# In[1]:
+
+
+import pandas as pd
+
+
+# In[2]:
+
+
+from mrjob.job import MRJob
+
+
+# In[6]:
+
+
+get_ipython().run_cell_magic('file', 'qwe_1.py', '\nfrom mrjob.job import MRJob\nimport re\n\nclass wordcount(MRJob):\n    @staticmethod\n    def mapper2(identifier, line):\n        words=[]\n        for i in identifier.lower().split():\n            words.append(i)\n            #uniquewords=set(words)\n            average=len(words)/10000\n        return average\n    \n    def mapper(self, _, line):\n        row=line.split(\',\')\n        identifier=row[4]\n        z=identifier.split(" ")\n        avg=self.mapper2(identifier,line)\n        #worz=len(z)\n        #worz=len(identifier.split())\n        yield "Average",avg\n\n    def reducer(self,key, values):\n        yield key,sum(values)\n        \n        \nif __name__ == \'__main__\':\n    wordcount.run()')
```

Comment:
this is the important part of your code, and it's very hard to read.  -1 for not properly parsing the CSV